html/elements/font.json

```diff
@@ -0,0 +1,57 @@
+{
+  "html": {
+    "elements": {
+      "font": {
```

Comment:
Does this need the 3 attributes color, face, size?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font

Reply:
Yes, fixed.